<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="content_plot">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>15</y>
     <width>650</width>
     <height>700</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>300</width>
     <height>700</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="genethic_algorithm">
    <attribute name="title">
     <string>Genethic algorithm</string>
    </attribute>
    <widget class="QLabel" name="label_number_of_chromosomes">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Число хромосом</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="khvan">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Костина функция</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="shishkina">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>470</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Дианина функция</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rosenbrock">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Розенброк</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_number_of_individuals">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of individuals</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_number_of_generations">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of generations</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="number_of_chromosomes">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="number_of_generations">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="number_of_individuals">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="lauch_button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>590</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>15</y>
     <width>270</width>
     <height>700</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>268</width>
      <height>698</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="output">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>270</width>
       <height>700</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
